### DISCUSSION TITLE
Is there a way to enable syntax highlighting for rendered markdown cells?

### DISCUSSION THREAD

**Discussion by codyish** [microsoft/vscode-jupyter#9113] [Questions and Answers] [answered]
Syntax highlighting using ````python```` looks correct while I'm editing a markdown cell (and if I open the same notebook in JupyterLab directly) but disappears when I run the cell to render it.

While editing
![image](https://user-images.githubusercontent.com/16812640/155567261-2f7d6e55-c2f6-4c49-83a5-03e7d5c8d5f6.png)

After rendering cell
![image](https://user-images.githubusercontent.com/16812640/155567382-831d1d0b-508f-4d64-8f4c-bb29e4b109bf.png)

The final comment from [this issue ](https://github.com/microsoft/vscode-jupyter/issues/1345#issuecomment-946202161) indicates it should work in markdown cells the same as in files but since it's not for me I assume I'm missing a config or setting somewhere. It does work correctly in standalone .md files.

**Comment by greazer** [comment, marked as answer]
Hm, I'm not seeing what you're seeing. Here's what I see with what I think is a similar example.

#### Markdown source
<img width="486" alt="image" src="https://user-images.githubusercontent.com/11685408/155586390-2ce23854-8d1f-4ad8-8212-d98b9541a075.png">

#### Markdown rendered
<img width="488" alt="image" src="https://user-images.githubusercontent.com/11685408/155586448-c6e6370e-95a7-4dcd-afe6-3f2be69982a9.png">


May be a bug here somewhere. What is the version of your VS Code, Python extension, Jupyter extension and PyLance extension?

**Reply by codyish** [reply]
Glad I might not be going crazy.

VS Code
Version: 1.59.1 (Universal)
Commit: 3866c3553be8b268c8a7f8c0482c0c0177aa8bfa
Date: 2021-08-19T11:53:52.479Z (6 mos ago)
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Darwin x64 21.2.0

Python extension
v2021.10.1365161279

Jupyter extension
v2021.8.1236758218

Pylance
v2022.1.3

**Reply by codyish** [reply]
### UPDATE - FIXED

I'm an idiot. I thought everything was auto-updating, but since you asked for versions I went ahead and double-checked that everything was current and I accidentally had VS Code in the downloads folder since moving everything to a new computer a few months ago so it wasn't updating. I moved it to the Applications folder and updated and now everything is working.
